2018/gulpfile.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const fs = require('fs');
+const fse = require('fse');
```

Comment:
what is fse ? I would like to be simple, fs is better, if you need promise, use fs.promises or bluebird instead.

Reply:
[`fs.promises`](https://www.npmjs.com/package/fs.promises) seems less well maintained than [`fse`](https://www.npmjs.com/package/fse) ?!

Reply:
nope, fs.promises is a new feature for node.js

Reply:
https://nodejs.org/api/fs.html#fs_fs_promises_api

Reply:
Gulp 3.9.1 is broken for Node.js 10 https://github.com/nodejs/node/issues/19786#issuecomment-379583525

I would need to update Gulp to the 4 beta to use `fs.promises`